### PR TITLE
Fixing method and tests to work on Linux and Windows OS

### PR DESCRIPTION
**Pull request by sphinix27**
Fixing tests to work on both OS windows or linux.

### CODE REVIEW

File: src/main/java/org/minions/devfund/abel/WhackAMole.java [hunk 1]
```diff
@@ -125,7 +125,7 @@ public String printGridToUser() {
     public String printGrid() {
         StringBuilder stringBuilder = new StringBuilder();
         for (char[] row : this.moleGrid) {
-            stringBuilder.append(String.format("%s%s", String.valueOf(row), System.lineSeparator()));
+            stringBuilder.append(String.format("%s%n", String.valueOf(row)));
```
Comment (carledriss) [inline src/main/java/org/minions/devfund/abel/WhackAMole.java:128]: I think you can fix the problem only using the following code.
```java
stringBuilder.append(row).append("\n");
```

**Review by carledriss — CHANGES_REQUESTED** [review]
line 109 is failing on Windows compilation
use the correct double quotes.

File: src/main/java/org/minions/devfund/abel/WhackAMole.java (hunk 1 above)
Comment (asalazarsl) [reply src/main/java/org/minions/devfund/abel/WhackAMole.java]: Windows:
image
Linux
image

I check on both OS.
I can use "\n" but I think that is just hardcode for windows.

Comment (sphinix27) [reply src/main/java/org/minions/devfund/abel/WhackAMole.java:128]: Windows:
image
Linux:
image

I can use "\n" but that is just hardcode and only works on Unix based.
https://stackoverflow.com/a/31981167

**Review by carledriss — APPROVED** [review]
LGTM! 👍

**Pull request merged**